### PR TITLE
[WIP] QA feedback from Ornela

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/6130

### CODE REVIEW

File: adoc/retail_chap_install.adoc [hunk 1]
```diff
@@ -128,16 +128,16 @@ Channel overview required for {smr} functionality:
     - SUSE Manager 3.2 Proxy for Retail Update
 ----
 
-After you have synchronized the required SUSE channels, create channels for custom software.
+After you have synchronized the required SUSE channels, create channels for any custom software.
 Channels are used to provide custom software for OS image building.
-For instructions see the xref:reference-webui-channels.adoc[Software Channels Reference]
+For more information on software channels, see xref:reference-webui-channels.adoc[Software Channels Reference]
```
Comment (keichwa) [inline adoc/retail_chap_install.adoc]:
```suggestion
For more information on software channels, see xref:reference-webui-channels.adoc[Software Channels Reference].
```
(add dot at sentence end)

Comment (keichwa) [inline adoc/retail_chap_install.adoc]:
```suggestion
For more information on software channels, see xref:reference-webui-channels.adoc[Software Channels Reference].
```
(add dot at sentence end)

Comment (keichwa) [inline adoc/retail_chap_install.adoc]:
```suggestion
For more information on software channels, see xref:reference-webui-channels.adoc[Software Channels Reference].
```
(add dot at sentence end)

**Comment by keichwa** [comment]
NIce dupes...  Now they show all my click attempts...

**Comment by keichwa** [comment]
Ornela Marić <notifications@github.com> writes:

> 0rnela approved this pull request.
>
> Looks good, thanks.

ATM, the github UI does not like me...

One final dot is missing:

- For more information on software channels, see xref:reference-webui-channels.adoc[Software Channels Reference]
+ For more information on software channels, see xref:reference-webui-channels.adoc[Software Channels Reference].

ANd then I'm wondering whether we'd better use title style for titles?
For example:

- == Branch server network configuration
+ == Branch Server Network Configuration

Otherwise: approved :)

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg)

**Comment by Loquacity** [comment]
> ANd then I'm wondering whether we'd better use title style for titles? For example: - == Branch server network configuration + == Branch Server Network Configuration

I just went and changed them all to sentence because I thought that was house style?

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

>> ANd then I'm wondering whether we'd better use title style for titles?
> For example: - == Branch server network configuration + == Branch Server
> Network Configuration
>
> I just went and changed them all to sentence because I thought that was
> house style?

The SUSE Documentation Team uses title style for all titles (chapter,
section, procedure, warnings).  In the past, the style guide initially
maintained by Rebecca Walter also wanted us (esp. YaST developers) to
use title style for button labels, selection list entries, etc.—we no
longer do this, AFAIK.

(the initial spacewalk upstream team sometimes had other ideas about
style, often not bad but a little bit special.  For example, down-casing
links such as "create new key" in the upper right corner of the Web UI.)

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg)

**Comment by Loquacity** [comment]
> Lana Brindley <notifications@github.com> writes:
> > ANd then I'm wondering whether we'd better use title style for titles? For example: - == Branch server network configuration + == Branch Server Network Configuration I just went and changed them all to sentence because I thought that was house style?
> The SUSE Documentation Team uses title style for all titles (chapter, section, procedure, warnings). In the past, the style guide initially maintained by Rebecca Walter also wanted us (esp. YaST developers) to use title style for button labels, selection list entries, etc.—we no longer do this, AFAIK. (the initial spacewalk upstream team sometimes had other ideas about style, often not bad but a little bit special. For example, down-casing links such as "create new key" in the upper right corner of the Web UI.)

Oh, I've misunderstood somewhere along the line, then. OK, let's address this in the next revision.
L

**Pull request merged**